src/compas/data/encoders.py

```diff
@@ -40,14 +40,16 @@
     numpy_support = False
 
 
-def cls_from_dtype(dtype):  # type: (...) -> Type[Data]
+def cls_from_dtype(dtype, mro=None):  # type: (...) -> Type[Data]
```

Comment:
i would not call this "mro". method resolution order refers to the mechanism that Python uses to determine which method should be called in a multiple inheritance scenario. but that is not what we do here...

Reply:
indeed, we use "inheritance" then